static/app/views/userFeedback/index.tsx

```diff
@@ -155,6 +169,17 @@ const Header = styled('div')`
   margin-bottom: ${space(2)};
 `;
 
+const PageFilters = styled('div')`
+  display: grid;
+  gap: ${space(1)};
+  grid-template-columns: minmax(0, 1fr) minmax(0, 1fr) max-content;
+  margin-bottom: ${space(2)};
+
+  @media (max-width: ${p => p.theme.breakpoints[0]}) {
+    grid-template-columns: 1fr;
+  }
+`;
```

Comment:
Should this wrapper be a shared component? Or do you expect that each product page will need to have a slightly different implementation?

Reply:
I think each product page will be having a slightly different implementation, especially those that have a search bar or other toggles/filters in the header. 

I think if there are more instances of this exact same css for this component though I will abstract it out.